toolkit/scss/forms/_validation.scss

```diff
@@ -53,7 +53,7 @@
 }
 
 .validation-message {
-  display: block;
+  display: inline-block;
```

Comment:
It'd be worth using the [inline-block shim](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_shims.scss#L17) from the govuk toolkit here.